UPCarouselFlowLayoutDemo.xcodeproj/project.pbxproj

```diff
@@ -402,11 +419,13 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = X4EYDGL7NV;
```

Comment:
It looks like there previously wasn't a development team set, should this be here?